unitTests/UnitTest11.cpp

```diff
@@ -1,179 +1,79 @@
-//
-// Created by Anders Cedronius on 2019-12-05.
-//
+#include <gtest/gtest.h>
+
+#include <memory>
+
+#include "ElasticFrameProtocol.h"
+#include "UnitTestHelpers.h"
+
+#define MTU 1456 //SRT-max
```

Comment:
Could this be moved into UnitTestHelpers.h and that header be included in all unit test cpp files?

Reply:
Sure!